cms/templates/emails/user_task_complete_email.txt

```diff
@@ -19,9 +19,11 @@ ${_("Errors:")}
   * ${error}
 % endfor
 
+% if olx_validation_errors.get('warnings'):
```

Comment:
Why have we added this check?

Reply:
If there are no warnings, we do not want to display `Warnings:` heading. 
```
Errors:
 * E1
 * E2

Warnings:
```